cromulent/data/crm_vocab.tsv

```diff
@@ -399,7 +399,7 @@ geo:SP5_Geometric_Place_Expression	class	Geometry	Geometry	Scope note: This clas
 geo:SP4_Spatial_Coordinate_Reference_System	class	CoordinateSystem	Spatial Coordinate Reference System	Scope note: This class compromises systems that are used to describe locations in a SP3 Reference Space (S). An instance of SP4 Spatial Coordinate Reference System is composed of two parts: The first is a Coordinate System which is a set of coordinate axes with specified units of measurement and axis directions. The second part is a set of reference features at rest in the Reference Space it describes in the real world that relate the Coordinate System to real world locations (U) and fix it with respect to the reference object of its Reference Space.	E29_Design_or_Procedure	1
 ore:Aggregation	class	Aggregation	Aggregation	ORE Aggregation is a conceptual collection of resources.	E89_Propositional_Object	1
 ore:Proxy	class	Proxy	Proxy	Proxy for a resource within the context of an Aggregation.	E36_Visual_Item	1
-P180_has_currency	property	currency	has currency	This property establishes the relationship between an instance of E97 Monetary Amount and the currency that it is measured in.	P91_has_unit	E97_Monetary_Amount	E98_Currency	P180i_was_currency_of	10000	1	0
+P180_has_currency	property	currency	has currency	This property establishes the relationship between an instance of E97 Monetary Amount and the currency that it is measured in.	P91_has_unit	E97_Monetary_Amount	E98_Currency		10000	1	0
```

Comment:
Can you remove this change please. Not sure why the inverse isn't automatically added, but it needs to be there.